_release_common.sh

```diff
@@ -41,6 +41,17 @@ function get_release_year {
 	echo "$(_get_product_version "${1}")" | cut -d '.' -f 1
 }
 
+function has_ssh_connection {
```

Comment:
@kiwm, move this function to `_common.sh` and make this in a single commit. :wink: